modules/seqexec/web/server/src/main/scala/seqexec/web/server/http4s/SeqexecCommandRoutes.scala

```diff
@@ -173,8 +171,7 @@ class SeqexecCommandRoutes(auth:       AuthenticationService,
         Ok(s"Stopped from queue $qid")
 
     case POST -> Root / "execute" / ObsIdVar(oid) / PosIntVar(step) / ResourceVar(resource) as _ =>
-      // Just simulate a delay
-      IO.sleep(2.seconds) *>
+      se.configSystem(inputQueue, oid, step, resource) *>
```

Comment:
So we are not using the response. How is it sent to the client?
How could the client know that a response is correlated with this request?

Reply:
I'll create a Jira task to add the `clientId`